pkg/crypto/eip712/typeddata.go

```diff
@@ -7,15 +7,15 @@ package eip712
 import (
 	"fmt"
 
-	"github.com/ethereum/go-ethereum/signer/core"
+	"github.com/ethereum/go-ethereum/signer/core/apitypes"
```

Comment:
How does this relate to the m1 usb?

Reply:
So the dep in question is `github.com/karalabe/usb` which is an indirect dependency because of `go-ethereum`. If we update `go-ethereum`, this package was changed and we need to update.

So seems like it will be better to do this in the next release as the RC is already out. I will hold this merge till we tag the next release.